builtin/game/chatcommands.lua

```diff
@@ -487,6 +487,26 @@ core.register_chatcommand("deleteblocks", {
 	end,
 })
 
+core.register_chatcommand("fixlight", {
+	params = "(here [radius]) | (<pos1> <pos2>)",
+	description = "Removes lighting bugs in the area between pos1 and pos2 "
```

Comment:
Perhaps better to not mention lighting bugs, instead more precise to say 'updates lighting ..' or something.